Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -9191,8 +9191,10 @@ void WebPageProxy::createNewPage(IPC::Connection& connection, WindowFeatures&& w
         configuration->setOpenedSite(site);
     } else {
         configuration->setOpenerInfo(std::nullopt);
-        configuration->setOpenedSite(WebCore::Site(request.url()));
-        if (openedBlobURL)
+        WebCore::Site openedSite { navigationAction->request().url() };
```

Comment:
`request.url()` seems to be empty here, but at the same time I'm not sure if changing the openedSite to `navigationAction->request().url()` is the right move..